.changes/unreleased/tinymce-TINY-11661-2024-12-23.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Not specifying a notification type (or specifying incorrect one) now defaults to 'info'
```

Comment:
We have a rule to add a `.` at the end of the log items so making it a sentence.